### PR TITLE
Creating CommandBuilders

### PR DESCRIPTION
**Pull request by shacharPash**

### CODE REVIEW

File: src/NRedisStack/Bloom/BloomCommands.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public BloomCommands(IDatabase db)
         /// <remarks><seealso href="https://redis.io/commands/bf.add"/></remarks>
         public bool Add(RedisKey key, RedisValue item)
         {
-            return _db.Execute(BF.ADD, key, item).ToString() == "1";
+            return _db.Execute(BloomCommandBuilder.Add(key, item)).ToString() == "1";
```
Comment (chayim) [inline src/NRedisStack/Bloom/BloomCommands.cs:24]: Yes

File: src/NRedisStack/Bloom/BloomSerializer.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                throw new ArgumentOutOfRangeException(nameof(items));
+
+            List<object> args = new List<object> { key };
+
+            foreach (var item in items)
+            {
+                args.Add(item);
+            }
+
+            return new SerializedCommand(BF.MADD, args);
```
Comment (slorello89) [inline src/NRedisStack/Bloom/BloomSerializer.cs:53]: Send args to an array so that it's interpreted correctly by the params macro

**Comment by chayim** [comment]
@shacharPash I think this PR should include changes to the sparse README. It's time to show users how they can use this.

File: src/NRedisStack/Bloom/BloomCommandBuilder.cs [hunk 3]
```diff
@@ -0,0 +1,95 @@
+using NRedisStack.Literals;
+using NRedisStack.RedisStackCommands;
+using StackExchange.Redis;
+namespace NRedisStack
+{
+
+    public static class BloomCommandBuilder
```
Comment (chayim) [inline src/NRedisStack/Bloom/BloomCommandBuilder.cs:7]: IMHO comment the class too. Let's explain to users why this exists, why it's an issue.

**Comment by chayim** [comment]
@leibale I asked Shachar to document.. anything else from your side? From a logic/impl perspective it looks sane to me.

**Pull request merged**